augur/config.py

```diff
@@ -183,7 +217,36 @@
                 "port": 51200,
                 "switch": 1,
                 "workers": 1
-            }
+            },
```

Comment:
These are needed when we relesae these workers, and updating configs once installed is another step. For now they are harmless.

Reply:
Just leave them please.